lib/orm/apollo_schemer.js

```diff
@@ -157,8 +161,10 @@ var schemer = {
         else if(typeof fieldob == 'object'){
             return fieldob.type;
         }
-        else
+        else {
+            throw("Uknown field '" + fieldname + "'");
             return undefined;
```

Comment:
Shouldn't this throw a message something like the following?

```
throw("Field type not defined for '" + fieldname + "'");
```

Could you please explain the scenario where an `Unknown field` error message would be appropriate?